src/Mod/PartDesign/Gui/Workbench.cpp

```diff
@@ -451,14 +451,12 @@ Gui::MenuItem* Workbench::setupMenuBar() const
           << "Separator"
           << "Part_CheckGeometry"
           << "Separator"
+          << "PartDesign_InvoluteGear"
           << "PartDesign_Sprocket";
 
     // For 0.13 a couple of python packages like numpy, matplotlib and others
     // are not deployed with the installer on Windows. Thus, the WizardShaft is
     // not deployed either hence the check for the existence of the command.

```

Comment:
Is this comment still necessary ?

Reply:
Yes. It applies to the shaft wizard, which indeed required some non-standard python modules. For some reason the gears sneaked in between the comment and the code it concerned. Now the commend is again correctly placed above the code it concerns.